src/wwwroot/app/views/reports/vacancies-report/vacancies.report.controller.js

```diff
@@ -171,9 +171,9 @@ export default function VacanciesReportController(
    }
 
    function isEqualLocations(user) {
-      return vm.vacanciesReportParametrs.locationIds.length ?
-         vm.vacanciesReportParametrs.locationIds.includes(user.cityId) :
-         true;
+      return isEmpty(vm.vacanciesReportParametrs.locationIds) ?
+         true :
+         vm.vacanciesReportParametrs.locationIds.includes(user.cityId);
```

Comment:
it is equals to:
isEmpty(vm.vacanciesReportParametrs.locationIds) || vm.vacanciesReportParametrs.locationIds.includes(user.cityId);